facets_mcp/tools/deploy_module.py

```
@@ -224,21 +229,24 @@ def test_already_previewed_module(project_name: str, intent: str, flavor: str, v
 
 
 @mcp.tool()
-def check_deployment_status(cluster_id: str, deployment_id: str, wait: bool = False, timeout_seconds: int = 300,
+def check_deployment_status(cluster_id: str, release_trace_id: str, wait: bool = False, timeout_seconds: int = 300,
```

Pylint (3.3.7)

[refactor] Too many return statements (7/6)

(R0911)